docs/Webpack.md

```diff
@@ -3,11 +3,11 @@ id: webpack
 title: Using with webpack
 ---
 
-Jest can be used in projects that use [webpack](https://webpack.js.org/) to manage assets, styles, and compilation. webpack _does_ offer some unique challenges over other tools because it integrates directly with your application to allow managing stylesheets, assets like images and fonts, along with the expansive ecosystem of compile-to-JavaScript languages and tools.
+Jest can be used in projects that use [webpack](https://webpack.js.org/) to manage assets, styles, and compilation. webpack _does_ offer some unique benefits over other tools because it integrates directly with your application to allow you the ability to manage assets like images, fonts, and stylesheets, and a host of compile-to-JavaScript languages and tools as part of webpack's expansive ecosystem. 
 
 ## A webpack example
 
-Let's start with a common sort of webpack config file and translate it to a Jest setup.
+Let us demonstrate how to configure Jest so that it can be used with webpack. We'll start with an example webpack `.config` file.
```

Comment:
```suggestion
Let us demonstrate how to configure Jest so that it can be used with webpack. We'll start with an example `webpack.config.js` file.
```